ExoDiPhotonAnalyzer/test/submit_crab_cfg.py

```diff
@@ -520,6 +520,91 @@
   for ids in ilist:
     cmssw_base = os.getenv("CMSSW_BASE")
     datasetID = ids.replace('/', '', 1).replace('/', '_', 1)
+f do2017signal:
```

Comment:
There are two syntax errors here:

-- `f` should be `if`
-- `do2017signal` should be defined at the top of this file. Conventionally, the default value is `False` so that one doesn't inadvertently re-submit jobs. Then you can uncomment your additions below.